lib/vrl/cli/src/lib.rs

```diff
@@ -6,16 +6,16 @@ pub use cmd::{cmd, Opts};
 
 #[derive(thiserror::Error, Debug)]
 pub enum Error {
-    #[error("io error")]
+    #[error("io error: {}", .0)]
     Io(#[from] std::io::Error),
 
-    #[error("parse error")]
+    #[error("{}",.0)]
```

Comment:
```suggestion
    #[error("{}", .0)]

Reply:
Also, does it make sense to not give it the "parse error" prefix anymore?  I have no idea what the internal string will say, but my wild guess is that it's parser specific i.e. written for a person who knows its coming from a parser, which wouldn't translate well here?

Reply:
Ah, sorry, I should have provided more context there. The parse errors are meant for end-users. They look like:

```
error[E105]: call to undefined function
  ┌─ :2:1
  │
2 │ unknown_function(.foo, .bar)
  │ ^^^^^^^^^^^^^^^^
  │ │
  │ undefined function
  │ did you mean "parse_duration"?
  │
  = learn more about error code 105 at https://errors.vrl.dev/105
  = see language documentation at https://vrl.dev
```

Reply:
I added a comment in 9d9e17c